packages/manager/modules/telecom-dashboard/src/index.js

```diff
@@ -0,0 +1,73 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'ng-at-internet';
+import 'at-internet-ui-router-plugin';
+import '@ovh-ux/ng-uirouter-title';
+import core from '@ovh-ux/manager-core';
+import tuc from '@ovh-ux/telecom-universe-components';
```

Comment:
it's better to inject using ```import foo from 'bar'``` right? so bar can change his module name without having to patch it here
maybe you want a more explicit name instead of tuc ?

Reply:
It's not.

In this context here we are dealing with a `peerDependency`. which implies that at the time you were  importing it, it will not be resolved.

Reply:
@antleblanc yep as discussed it will cause trouble for UMD target (since everything will be bundled in dist)